docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/posthog/page.mdx

---
title: 'PostHog'
description: 'Ingest PostHog product analytics events into ClickHouse with GlassFlow'
---
import { Callout } from 'nextra/components'

# PostHog

PostHog is an open-source product analytics platform that captures user events, sessions, and feature usage in real time. It enables teams to analyze behavior, run experiments, and build data-driven product insights without relying on third-party SaaS tools.

## Using PostHog with GlassFlow

GlassFlow can ingest PostHog event data into ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, joins, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** PostHog connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** PostHog connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
